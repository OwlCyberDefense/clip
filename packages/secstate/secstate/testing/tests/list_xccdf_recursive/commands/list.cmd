/usr/sbin/secstate list -r
