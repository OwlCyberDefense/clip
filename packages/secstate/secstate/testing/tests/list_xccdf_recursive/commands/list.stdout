Benchmark - ID: PassComp, Title: 'Password Complexity', Profile: None
	profile - id: emptyprofile, title: 'an empty profile'
	profile - id: none, title: 'none'
	Group - ID: PassComp-G-2-2, Title: 'Password'
		Group - ID: PassComp-G-2-3, Title: 'Password Complexity'
			Rule - ID: PassComp-R-2-1, Title: 'Lowercase'
			Rule - ID: PassComp-R-2-2, Title: 'Min. Length'
			Rule - ID: PassComp-R-2-3, Title: 'Numeric'
			Rule - ID: PassComp-R-2-4, Title: 'Special'
			Rule - ID: PassComp-R-2-5, Title: 'Uppercase'
